reservation-service/src/test/java/com/incubator/dbs/reservationservice/intergration/GuestServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.incubator.dbs.reservationservice.intergration;
+
+import com.incubator.dbs.reservationservice.model.dto.UserInfoResponseDTO;
+import java.net.URI;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.cloud.contract.stubrunner.spring.AutoConfigureStubRunner;
+import org.springframework.cloud.contract.stubrunner.spring.StubRunnerPort;
+import org.springframework.cloud.contract.stubrunner.spring.StubRunnerProperties;
+import org.springframework.http.HttpMethod;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.RestTemplate;
+
+@RunWith(SpringRunner.class)
+@AutoConfigureStubRunner(stubsMode = StubRunnerProperties.StubsMode.LOCAL, ids = "com.incubator.dbs:guest-service")
```

Comment:
Is this working fine? What if guest service creates jar file with different version? Will it be able to access correct/latest jar created for stub ?